config/initializers/trusted_proxies.rb

```diff
@@ -4,7 +4,7 @@ def trusted_proxy?(ip)
       if Rails.application.config.action_dispatch.trusted_proxies.nil?
         super
       else
-        Rails.application.config.action_dispatch.trusted_proxies.any? { |proxy| proxy === ip }
+        Rails.application.config.action_dispatch.trusted_proxies.any? { |proxy| proxy.include?(ip) }
```

Comment:
Why? This is not right. `proxy` is IPAddr, not an array.

Reply:
sorry. I miss-read this.